avs-manager/src/UniFiAVSManager.sol

```diff
@@ -227,22 +267,23 @@ contract UniFiAVSManager is IUniFiAVSManager, UniFiAVSManagerStorage, UUPSUpgrad
         uint256 validatorCount = validatorPubkeys.length;
 
         for (uint256 i = 0; i < validatorCount; i++) {
-            ValidatorData storage validator = $.validators[validatorPubkeys[i]];
+            bytes memory validatorPubkey = validatorPubkeys[i];
+            ValidatorData storage validator = $.validators[validatorPubkey];
 
             address operator = validator.operator;
 
             if (operator != msg.sender) {
```

Comment:
In a scenario:
- EigenPod is delegated to Operator A
- Operator A registers validator 0x123
- Eigenpod undelegates from Operator A, and delegates to operator B

0x123 is still registered to AVS and validator.operator is still A, operator B can't deregister the validator from the AVS, and might potentially get slashed in the future.

Is this a valid scenario?